sycl/plugins/opencl/pi_utils.hpp

```diff
@@ -0,0 +1,79 @@
+//===--------- pi_utils.hpp - Plugin Utility Functions -------------------===//
```

Comment:
pi_utils.hpp will be removed. This code is available from unified runtime. Thanks.